packages/bpk-component-checkbox/stories.js

```diff
@@ -104,12 +57,7 @@ class StatefulCheckbox extends Component<
 
```

Comment:
Simplified the stateful example to use minimal lines of code and not worry about indeterminate.

I could make it even shorter with `useState`, but our React version doesn't support that.